kafka-client/src/main/resources/otel-jmx.config.yaml

```diff
@@ -0,0 +1,334 @@
+---
```

Comment:
this is the same as https://github.com/responsivedev/responsive-pub/blob/main/etc/otel-jmx.config.yaml right? Can you remove that file now?

Reply:
Yes, I will remove it once I migrate all the examples to use the builtin reporter